config/routes.rb

```diff
@@ -1,4 +1,8 @@
+require 'sidekiq/web'
+require 'sidekiq/cron/web'
+
 Rails.application.routes.draw do
+  mount Sidekiq::Web => '/sidekiq'
```

Comment:
It's fine for educational purposes. Please notice that this endpoint shouldn't be publicly accessible in production :wink:

Reply:
I have found this solution on sidekiq github wiki. Would that be enough?

```
authenticate :user, lambda { |u| u.admin? } do
  mount Sidekiq::Web => '/sidekiq'
end
```